apps/rush/src/utilities/PackageReviewChecker.ts

```diff
@@ -41,9 +39,18 @@ export default class PackageReviewChecker {
         const scope: string = Utilities.parseScopedPackageName(packageName).scope;
 
         // Make sure the scope isn't something like "@types" which should be ignored
-        if (!this._packageReviewConfiguration.ignoredNpmScopes.has(scope)) {
+        if (!this._rushConfiguration.approvedPackagesIgnoredNpmScopes.has(scope)) {
           // Yes, add it to the list if it's not already there
-          this._packageReviewConfiguration.addOrUpdatePackage(packageName, false, rushProject.reviewCategory);
+
+          // By default we put everything in the browser file.  But if it already appears in the
+          // non-browser file, then use that instead.
+          if (this._rushConfiguration.nonbrowserApprovedPackages.getItemByName(packageName)) {
+            this._rushConfiguration.nonbrowserApprovedPackages
+              .addOrUpdatePackage(packageName, rushProject.reviewCategory);
+          } else {
```

Comment:
wait, does this mean that the package should be in BOTH lists?? Also, won't this mean that if the package is only approved for non-browser it will constantly show up in the browser list... this logic seems odd